Doc/Tutorial/chapter_align.tex

```diff
@@ -224,14 +222,14 @@ \subsection{Multiple Alignments}
 
 If you wanted to read this in using \verb|Bio.AlignIO| you could use:
 
-%TODO - Replace the print blank line with print()?
 \begin{minted}{python}
-from Bio import AlignIO
+>>> from Bio import AlignIO
 
```

Comment:
This blank line needs to be removed or have ``>>>`` present if formatted as a docstring.